test/functional/ex_cmds/profile_spec.lua

```diff
@@ -23,6 +25,19 @@ local function assert_file_exists_not(filepath)
   eq(nil, lfs.attributes(filepath, 'uid'))
 end
 
+local function read_file(filepath)
```

Comment:
looks similar to `test/helpers.lua`. Perhaps that one could be changed to return `msg`, then we don't need this one?

Reply:
I had overlooked that.